react-frontend/src/components/Guides/CollaborationTools/collaborationTool.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+function CollaborationTool(props) {
```

Comment:
prefer arrow style functions of regular function declarations now-a-days for react components. This is straight out of the react docs they seldom use regular function expressions. 

`export const CollaborationTool = ({url, name, cost, platform, capacity}) => ( ...);`

Reply:
Changing over to arrow functions is a decet idea, though I'd recommend doing so for the whole codebase at once.

Reply:
not sure what you mean